Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -164,6 +168,20 @@ QObject* AzAssetBrowserWindow::createListenerForShowAssetEditorEvent(QObject* pa
     return listener;
 }
 
+void AzAssetBrowserWindow::resizeEvent(QResizeEvent* ev)
```

Comment:
nit: Prefer more descriptive variable names, this would ideally be `resizeEvent`